frontend/src/views/Game.tsx

```diff
@@ -457,6 +457,7 @@ function GamePage() {
                 onLanguageChange={setCurrentLanguage}
                 codeMap={defaultCodeList[0]}
                 defaultLanguage={currentLanguage}
+                defaultCode={null}
```

Comment:
This is the default code used if you already know what code you want to show, i.e. for the previewing of other people's code. Although for the game page we use codeMap for that, the codeMap types requires default code for all the languages (which is generated from the backend), while on the results page we only have the user's submission language, so basically this param is used for that. 

I couldn't figure out how to do implement optional props without some ESLint default props issues, so lmk if you know the solution for that :) (or if you had any other slightly better solutions, since ik this is a little hacky)

Reply:
I think this is alright, not too hacky - if this solves the issue fully, no problem let's go with this.